proxies-es6/demo.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+// Proxying a normal object
+var target = {};
+var handler = {
+  get(receiver, name = 'world') {
+    return `Hello, ${name}!`;
+  }
+};
+
+var p = new Proxy(target, handler);
+ChromeSamples.log(p.folks);
+
+// Proxying a function object
+function sum(a, b) {
+  return a + b;
+}
+
+var handler2 = {
+  apply: function(target, thisArg, argumentsList) {
+    console.log(`Calculate sum: ${argumentsList}`);
```

Comment:
This line isn't being logged when I test in Chrome Canary 49.0.2622.0:

![image](https://cloud.githubusercontent.com/assets/1749548/12356233/7d5977b8-bb6f-11e5-9a43-59fa36440f78.png)